ramls/patron-notice-policy.json

```diff
@@ -207,74 +157,31 @@
             "properties": {
               "sendHow": {
                 "type": "string",
-                "description": "Defines how notice should be sent, before, after or upon",
+                "description": "Defines how notice should be sent, after or upon",
```

Comment:
There should probably be dash or colon after "sent", otherwise it's a bit confusing.

Reply:
Fixed.